Python/PRP/pyCreatableConvert.cpp

```diff
@@ -377,7 +379,7 @@ plCreatable* IConvert(pyCreatable* pCre)
     else if (Py_TYPE(pCre) == &pyResponderModifier_Type) return dynamic_cast<plCreatable*>(reinterpret_cast<plResponderModifier*>(pCre->fThis));
     //else if (Py_TYPE(pCre) == &pyAxisAnimModifier_Type) return dynamic_cast<plCreatable*>(reinterpret_cast<plAxisAnimModifier*>(pCre->fThis));
     //else if (Py_TYPE(pCre) == &pyLayerLightBase_Type) return dynamic_cast<plCreatable*>(reinterpret_cast<plLayerLightBase*>(pCre->fThis));
-    //else if (Py_TYPE(pCre) == &pyFollowMod_Type) return dynamic_cast<plCreatable*>(reinterpret_cast<plFollowMod*>(pCre->fThis));
+    else if (Py_TYPE(pCre) == &pyFollowMod_Type) return dynamic_cast<plCreatable*>(reinterpret_cast<plFollowMod*>(pCre->fThis));
```

Comment:
I had a feeling I was forgetting one of the several places to which we add a new object.